### PR TITLE
Adding lmx9838 footprint to RF_Modules.pretty library

### PR DESCRIPTION
**Pull request by gsgill112**
Pl. help 

Violating Rule 7.4
    Fabrication layer requirements
    Second Reference Designator missing
     - Add RefDes to F.Fab layer with '%R'

!!

### CODE REVIEW

**Comment by Shackmeister** [comment]
Try to have a look at some other footprints regarding the error 
Also please reset the scale, offset and rotation and use stepup for freecad instead to align your 3d model

**Comment by jkriege2** [comment]
Hi!

I think `Digi_XBee_SMT` would be a good starting point ... some comments on this footpritn:
1. The name should be `Texas_LMX9838`
2. The text on F.Fab should be `%R`instead of `REF**` (note that it will still show `REF**`, as `%R` will be replaced with the REFDES)
3. Thickness of outline on F.Fab is OK, but looks nicer if it is 0.12mm or 0.1mm
4. THe drawing on the silkscreen should be visible after assembly and for SMD-parts you should not have silkscreen-drawings below.
5. Please draw the keepout-area onto Dwgs.User and add a text that this is a keepout-area (see `Digi_XBee_SMT`)
6. Please add some meaningful description to the description field and possibly some more meaningful keywords to `tags`. Also please add the URL of the datasheet to description.
7. When doint the 3D model (your other PR), please ensure that it is drawn in a way that it fits with offset (0,0,0) and scaling (1,1,1). StepUP will help you with this!

Best,
JAN

**Comment by gsgill112** [comment]
@Jan
Does this look Fine ?

I am really confused about your 2nd point.  Further I have studied the KLC
as well as went through other examples. Still can't kill the 7.4 Error :(

Regards
GurI

**Comment by gsgill112** [comment]
screen shot 2017-05-12 at 11 54 21 am

File: lmx9838.kicad_mod [hunk 1]
```diff
@@ -0,0 +1,86 @@
+(module lmx9838 (layer F.Cu) (tedit 591223DE)
+  (descr lmx9838.pdf)
+  (tags lmx9838)
+  (attr smd)
+  (fp_text user REF** (at 0 -4.25) (layer F.Fab)
```
Comment (poeschlr) [inline lmx9838.kicad_mod]: This is the reason for your 7.4 error. The user text should be %R not REF**

**Comment by poeschlr** [comment]
Sadly there are a lot of problems with this footprint.
The lines of the courtyard outline are not straight. (Make sure it has a clearance of 0.25mm compared to the package outline when you re draw it.)
The lines on the fab layer do not meet. (the ends on the corners do not have the same coordinates for two lines. The coordinates are not quite right see below)
The lines on fab should have a line width of 0.1mm
The silk screen outline is missing. (Width 0.12mm)
A pin 1 marker on Silk and Fab is missing.
Remove your current drawings from silk. **According to the datasheet it is not allowed to place silk below the part!** (Page 41)
Add a keepout marking on comments layer. See the new entry in the KLC (Rule 7.8) for details. (The outline of the keepout is the same as the body outline. Again see page 41)
screenshot from 2017-08-21 11-04-56

The measurements for the outline on fab are not quite right:
screenshot from 2017-08-21 11-35-59

**Comment by SchrodingersGat** [comment]
Hi @gsgill112   apologies for the inconvenience but we have marked this repository as read-only as we transition to the new repository for the v5 release.

Please resubmit this pull request to https://github.com/kicad/kicad-footprints

**Pull request closed without merging**